crates/metadata-server/src/lib.rs

```diff
@@ -517,39 +512,28 @@ impl JoinClusterHandle {
     }
 }
 
-/// Identifier to detect the loss of a disk.
-type StorageId = u64;
-
-#[derive(
-    Clone,
-    Copy,
-    Debug,
-    Eq,
-    Hash,
-    PartialEq,
-    serde::Serialize,
-    serde::Deserialize,
-    prost_dto::IntoProst,
-    prost_dto::FromProst,
-)]
-#[prost(target = "crate::grpc::MemberId")]
+type CreatedAtMillis = i64;
+
+/// A member of the replicated metadata store is identified by its `node_id` and a durable
+/// timestamp when it was first started.
```

Comment:
This looks very similar to StorageMarker<PlainNodeId>. it's probably still better to keep this as a different struct

Reply:
Yes, I would keep it separate for the time being.

Reply:
Ah true 🤦‍♂️. But this means use shouldn't change node name ever after provisioning

Reply:
Yes, changing node names is not a good idea.